package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "commercetools-nodejs-boilerplate",
+  "version": "1.0.0-beta.1",
```

Comment:
I'm not sure it makes much sense to publish this boilerplate to npm. If so, we should add `private: true`.